MetafileImporter/qSlicerMetafileImporterModule.h

```diff
@@ -34,6 +36,9 @@ qSlicerMetafileImporterModule
   : public qSlicerLoadableModule
 {
   Q_OBJECT
+#ifdef Slicer_HAVE_QT5
```

Comment:
Can't we just use (QT_VERSION < QT_VERSION_CHECK(5, 0, 0)) everywhere? If it works then let me know and then we should probably change this in the module templates as well.

Reply:
I guess it is temporary anyway (as we'll switch to Qt5 permanently) and it may be easier to find Slicer_HAVE_QT5 strings than QT_VERSION_CHECK. So, let's just use this everywhere.